<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroEditor</class>
 <widget class="QDialog" name="MacroEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macro Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Account:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_accounts"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_add_account">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>134</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Macros</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_add_macro">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_edit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_remove">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>215</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_backup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Backup to file</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_restore">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Restore backup</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>comboBox_accounts</zorder>
  <zorder>listWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_add_macro</zorder>
  <zorder>pushButton_remove</zorder>
  <zorder>pushButton_edit</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>buttonBox</zorder>
  <zorder>pushButton_backup</zorder>
  <zorder>pushButton_restore</zorder>
  <zorder>pushButton_add_account</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MacroEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MacroEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
